avalanche/benchmarks/scenarios/rl_scenario.py

```diff
@@ -9,14 +9,108 @@
 # Website: avalanche.continualai.org                                           #
 ################################################################################
 """Reinforcement Learning scenario definitions."""
-from typing import Union, Sequence
+from avalanche.benchmarks.scenarios import CLExperience, ExperienceAttribute, CLScenario, EagerCLStream
+from typing import Callable, List, Union, Dict
+import numpy as np
+import torch
+import random
 
 try:
-    from gym import Env
+    from gym import Env, Wrapper
 except ImportError:
-    # empty class to make sure everything below works without changes
+    # empty classes to make sure everything below works without changes
     class Env:
         pass
+    class Wrapper:
+        pass
+
+
+class RLExperience(CLExperience):
+
+    def __init__(self, env: Union[Env, Callable[[], Env]], n_envs: int = 1, task_label: int = None, current_experience: int = None, origin_stream=None):
+        # current experience and origin stream are set when iterating a CLStream by default
+        super().__init__(current_experience, origin_stream)
+        self.env = env
+        self.n_envs = n_envs
+        # task label to be (optionally) used for training purposes
+        self.task_label = ExperienceAttribute(task_label, use_in_train=True)
+
+    @property
+    def environment(self) -> Env:
+        # supports dynamic creation of environment, useful to instantiate envs for evaluation
+        if not isinstance(self.env, Env):
+            return self.env()
+        return self.env
+
+
+class RLScenario(CLScenario):
+
+    def __init__(self, envs: List[Env],
+                 n_experiences: int,
```

Comment:
Do we need the `n_experiences` arguments? Since the constructor takes a list of environments I don't see a need for it. The user can manipulate the list easily to modify the stream before calling the constructor.